it_zwierzak/homepage/urls.py

```diff
@@ -8,6 +8,6 @@
     path('statute/', views.statute, name='statute'),
     path('contact/', views.contact, name='contact'),
     path('advice/', views.advice, name='advice'),
-    path('login/', views.login, name='login'),
+    path('login/', views.log_in, name='login'),
     path('register/', views.register, name='register'),
```

Comment:
login, czy rejestracja najczęściej są w osobnej apce djangowej która typowo nazywa się `accounts, user, auth ...`

Reply:
czyli lepiej zrobić nową aplikację startapp user i tam przenieść logowanie i rejestrację?

Reply:
byłoby fajnie mieć rzeczy dotyczące autentykacji mieć w jednym miejscu, generalnie apki djangowe powinny być podzielone tak by w każdej z nich znajdowały się rzeczy które niejako są logicznie spójne

Reply:
dobrze, to tak zrobię